app/models/course/assessment/submission.rb

```diff
@@ -91,6 +91,22 @@ def auto_grade!
     AutoGradingJob.perform_later(self)
   end
 
+  # Return the maximum step that the current submission can reach.
+  #
+  # @return [Integer] the maximum step.
+  def max_step
```

Comment:
Because ordinal numbers can be used to identify *anything*; the number itself doesn't tell the user what it means.